fenicsprecice/adapter_core.py

```diff
@@ -427,34 +427,35 @@ def get_forces_as_point_sources(fixed_boundary, function_space, data):
 
     Returns
     -------
-    x_forces : list
-        Dictionary carrying X component of forces with reference to each point on the coupling interface.
-    y_forces : list
-        Dictionary carrying Y component of forces with reference to each point on the coupling interface.
+    forces : list
+        Multiple lists carrying components of forces with reference to each point on the coupling interface.
```

Comment:
I don't think that this update of the documentation is correct. You are still returning `d` lists below.

Reply:
Yes I am returning `dims` number of lists so is the description not correct? I mean to say that there are several lists returned. So something like:
```
Multiple lists carrying components of forces with reference to points on the coupling interface.
```

Reply:
The text is fine, but I still find the `forces: list` confusing. Not sure how to actually document multiple return values, if some of them are optional. Maybe something like

```
forces_x : list
    ....
forces_y: list
    ....
forces_z: list
    (optional, only returned, if 3D) ....
```